company_linkage/parat_data_dag.py

```diff
@@ -351,22 +351,25 @@
             create_disposition="CREATE_IF_NEEDED",
             write_disposition="WRITE_TRUNCATE"
         )
-        pop_descriptions = PythonOperator(
-            task_id="populate_column_documentation_for_" + table,
-            op_kwargs={
-                "input_schema": f"{os.environ.get('DAGS_FOLDER')}/schemas/parat/{table}.json",
-                "table_name": prod_table_name
-            },
-            python_callable=update_table_descriptions
-        )
         table_backup = BigQueryToBigQueryOperator(
             task_id=f"back_up_{table}",
             source_project_dataset_tables=[f"{staging_dataset}.{table}"],
             destination_project_dataset_table=f"{backups_dataset}.{table}_{curr_date}",
             create_disposition="CREATE_IF_NEEDED",
             write_disposition="WRITE_TRUNCATE"
         )
-        wait_for_checks >> copy_to_production >> pop_descriptions >> table_backup >> wait_for_copy
+        if table != "all_visualization_data":
```

Comment:
We don't have column documentation for this non-analyst-facing table, so skip the documentation population in its case